go.mod

```diff
@@ -110,24 +110,24 @@ require (
 	helm.sh/helm/v3 v3.12.2
 	istio.io/api v1.19.0-alpha.1.0.20230810203008-3cdd517bf131
 	istio.io/client-go v1.19.0-alpha.1.0.20230810203904-45e85278e0fc
-	k8s.io/api v0.28.0-beta.0
-	k8s.io/apiextensions-apiserver v0.28.0-beta.0
-	k8s.io/apimachinery v0.28.0-beta.0
-	k8s.io/apiserver v0.28.0-beta.0
-	k8s.io/cli-runtime v0.28.0-beta.0
-	k8s.io/client-go v0.28.0-beta.0
+	k8s.io/api v0.29.0-alpha.0
```

Comment:
I think we want to bump to v0.28.0. We only did the non-official release since we needed one of the changes they made in 1.28 in 1.19 and wanted to get early testing/merging